### PR TITLE
#170074178 Update the .env.sample and readme files

### PR DESCRIPTION
**Pull request by william000000**
#### What does this PR do?
- Update readme and .env.sample
#### Description of Task to be completed?
- Update readme
- Update .env.sample file
- Fix the test error

#### How should this be manually tested?
- You have to clone the repo `git clone https://github.com/andela/team-odd-bn-backend.git`
- Then After the repo cloned `cd project-folder` and run `npm i` to install all dependencies
- Go to my branch `git checkout accept-and-reject-request-169258642`
- Run app by `npm run dev-start`
- Open `Postman` app
- Navigate to `localhost:{server_port}/routepath`
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#170074178](https://www.pivotaltracker.com/story/show/170074178)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -10,8 +10,7 @@ FACEBOOK_APP_ID=YOUR_FB_APP_ID
 FACEBOOK_APP_SECRET=YOUR_FB_APP_SECRET
 GOOGLE_APP_ID=YOUR_GOOGLE_APP_ID
 GOOGLE_APP_SECRET=YOUR_GOOGLE_APP_SECRET
-MANAGER_EMAIL=YOUR_MANAGER_EMAIL
-MANAGER=MANAGER_NAME
 SUPER_ADMIN_EMAIL=SUPER_EMAIL
 SUPER_ADMIN_REAL_PASSWORD=PASSWORD
+PASSWORD_ENV=STRONG_PASSWORD
```
Comment (kipropbrian) [inline .env.example]: What is this for?

Comment (william000000) [reply .env.example]: It is a password we use for the test.

Comment (kipropbrian) [reply .env.example]: Should be TEST_PASSWORD_ENV

File: README.md [hunk 2]
```diff
@@ -43,9 +43,15 @@ In your terminal:
 
 2. For **Testing**: run `npm run test`
 
-3. For **Development**: run `npm run dev-start`
+3. Drop Tables run `npm run dropTables` to make sure you have update changes
```
Comment (victkarangwa) [inline README.md:47]: Thanks for this initiative @william000000

**Review by victkarangwa — APPROVED** [review]

**Pull request merged**